torchaudio_contrib/functional.py

```diff
@@ -47,6 +47,10 @@ def stft(signal, fft_len, hop_len, window,
     return spect
 
 
+def spectral_downmix(tensor, power=1.0):
```

Comment:
In general, I would prefer to add an option `axis=1` that defaults to the expected axis in the case of `(batch, channel, time)`. But why restrict to that? it might happen that some user has `(batch, other_stuff, channel, time)`.

Reply:
okay, yes. I agree

Reply:
Note, this is a general comment. If we add the `axis` option here a user would expect the same for all other functionals as well. But it should be fine, if we start with `spectral_downmix` and later add it to other functionals.